server/sonar-server/src/test/java/org/sonar/server/issue/ws/SearchActionMediumTest.java

```diff
@@ -698,22 +698,22 @@ private RuleDto newRule() {
     return rule;
   }
 
-  private void setDefaultProjectPermission(ComponentDto project) {
+  private void setDefaultProjectPermission(OrganizationDto organizationDto, ComponentDto project) {
     // project can be seen by anyone and by code viewer
     userSessionRule.login("admin").setGlobalPermissions(GlobalPermissions.SYSTEM_ADMIN);
     // TODO correctly feed default organization. Not a problem as long as issues search does not support "anyone"
     // for each organization
-    GroupPermissionChange permissionChange = new GroupPermissionChange(PermissionChange.Operation.ADD, UserRole.USER, new ProjectId(project), GroupIdOrAnyone.forAnyone("TODO"));
+    GroupPermissionChange permissionChange = new GroupPermissionChange(PermissionChange.Operation.ADD, UserRole.USER, new ProjectId(project), GroupIdOrAnyone.forAnyone(organizationDto.getUuid()));
     tester.get(PermissionUpdater.class).apply(session, asList(permissionChange));
   }
 
-  private void setProjectPermission(ComponentDto project, String... permissions) {
+  private void setProjectPermission(OrganizationDto organizationDto, ComponentDto project, String... permissions) {
```

Comment:
you're right. I added the new parameter without thinking it through. I'll remove it. It will simplify the code.